src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.data;
+
+/** String constants used in Servlets. */
+public final class Constants {
```

Comment:
Not sure if this will end up being a problem but I also have a class called "Constants"... Should we keep these classes separate (EventConstants and UserConstants), or just combine them??

Reply:
I think we can just combine them and the comment sections for the different types.